contribute/style-guides/e2e.md

```diff
@@ -58,11 +58,13 @@ Now that we have a `Page` called `Login` in our `Pages` const we can use that to
 
 The last step in our example is to use our `Login` page as part of a test. The `pageFactory` function we used before gives us two things: 
 - The `url` property is used whenever we call the `visit` function and is equivalent to the Cypress function [cy.visit()](https://docs.cypress.io/api/commands/visit.html#Syntax).
+> Best practices after calling visit is to always call `should` on a selector to prevent flaky tests where you try to operate on a selector that isn't ready. Read more about it [here](https://docs.cypress.io/guides/core-concepts/retry-ability.html#Commands-vs-assertions).  
```

Comment:
```suggestion
> Best practice after calling `visit` is to always call `should` on a selector to prevent flaky tests when you try to access an element that isn't ready. Read more about it [here](https://docs.cypress.io/guides/core-concepts/retry-ability.html#Commands-vs-assertions).  
```